crates/humanode-runtime/src/lib.rs

```diff
@@ -863,7 +816,7 @@ construct_runtime!(
         TokenClaimsPot: pallet_pot::<Instance3> = 11,
         TransactionPayment: pallet_transaction_payment = 12,
         Session: pallet_session = 13,
-        Offences: pallet_offences = 14,
+        HumanodeOffences: pallet_humanode_offences = 14,
```

Comment:
Reusing the pallet bumber might cause issues, use a different number

Reply:
https://github.com/humanode-network/humanode/pull/1291/commits/f970db1a7e547e20d015f4be55b3b11d865b3347